linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_login_defs/tests/wrong_value.fail.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 #
-# profiles = xccdf_org.ssgproject.content_profile_ospp
```

Comment:
This depends on var_accounts_user_umask. In some profiles, it's set to 027 and in other profiles it's set to 077, so in some profiles this test should pass.

Reply:
This is a good catch, cases like this are hard to handle automatically. I have exempted this rule's tests from the PR.